src/main/java/objective/SuspiciousFile.java

```diff
@@ -0,0 +1,2 @@
+// password = 1234
```

Comment:
Suspeita de credencial encontrada, tipo: Password

Se for um falso positivo, adicione o seguinte na mensagem do commit (amend) para que seja ignorada:
```
FalsoPositivo: a19e585c
```